### ISSUE TITLE
Block Break/Place Lag

### ISSUE THREAD

**Issue by ghost** [TTFTCUTS/ATG#10]
using forge 2296, minecraft 1.11.2 and using ATG 2.1.4. in some places, when i break any block or place anything on anything, it lags for about 0.1-0.2 second, yeah it's not problem but if i use it in my modpack it makes me wait about 3 seconds. i spent 2 hours to find which mod causes this. please help, thanks...
**Tests:**
It happens with ATG 2.1.4, CompatLayer 0.2.8.
It doesn't happens with ATG 2.0.1, CompatLayer 0.2.8.

**Comment by ghost** [comment]
and i'm aware of the version is experimental, i just wanted to help this mod be developed more.

**Comment by GirafiStudios** [comment]
Do you have any mods besides ATG?
ATG would most likely have nothing to do with the issue.

**Comment by ghost** [comment]
it doesn't happen on ATG 2.0.1.
I used 3 more mods (immersivecraft, compatlayer, xaero's minimap (fair play edition)).
i will test it without them right now, i will let you know.

**Comment by cjburkey01** [comment]
I am experiencing this as well. Using: 1.11.2-2.1.4

**Comment by ghost** [comment]
@cjburkey01 do you have any mods from my mods? immersivecraft, compatlayer and xaero's minimap ?

**Comment by cjburkey01** [comment]
@kenijey I have immersivecraft and compatlayer, not any others, I don't think.

**Comment by ghost** [comment]
yeah it happens only with CompatLayer. should i report it to McJty?

**Comment by kenijey** [comment]
btw, i told it to McJty.

**Comment by cjburkey01** [comment]
It may be Compatlayer, too, but something changed between version 2.0.1 and 2.1.4 of ATG. I just tested 2.0.1 and it functions fine. Also, where did you tell him? Did you create an issue at the [Compatlayer repository](https://github.com/McJty/compatlayer/issues)?

**Comment by TTFTCUTS** [comment]
No, it's me. It's the grass colour smoothing. I need to upload the version that's on forgecraft with the config option sometime. For some reason it's affecting some people with better hardware than I have *way* more than it's affecting me. I barely notice a difference despite relatively low base framerate but others get massive hangups.

**Comment by ghost** [comment]
kay, we will wait for a non-experimental version.

**Comment by McJty** [comment]
CompatLayer is a library. It is not even a mod. CompatLayer has *no* code that executes on its own so it cannot be the cause of this. That's literally impossible

**Comment by DropBear77** [comment]
dude, this is kinda game breaking issue for me. I get so bad lag when your mod is installed but i love it and would love to have it installed. If there is already a fix out there what the guys on forgecraft already use, please i bag you realeas it so everyone how has this issue is happy as a pig in shit and can play it

**Comment by ghost** [comment]
yeah it makes the game unplayable :/
